src/dioptra/restapi/v1/tags/schema.py

```diff
@@ -19,20 +19,83 @@
 
 from marshmallow import Schema, fields
 
+from dioptra.restapi.v1.groups.schema import GroupRefSchema
+from dioptra.restapi.v1.schemas import (  # generate_base_resource_schema,
+    BasePageSchema,
+    GroupIdQueryParametersSchema,
+    PagingQueryParametersSchema,
+    ResourceTypeQueryParametersSchema,
+    SearchQueryParametersSchema,
+)
+from dioptra.restapi.v1.users.schema import UserRefSchema
 
-class TagRefSchema(Schema):
+
+class TagMutableFieldsSchema(Schema):
+    """The fields schema for the mutable data in a Queue resource."""
+
+    name = fields.String(
+        attribute="name", metadata=dict(description="Name of the Queue resource.")
+    )
+
+
+# TagBaseSchema = generate_base_resource_schema("Tag") # Tag is not a resource.
+
+
+class TagBaseSchema(Schema):
```

Comment:
The current resolution to my comment in #394.  https://github.com/usnistgov/dioptra/pull/394#discussion_r1496467434